spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfiguration.java

```diff
@@ -81,6 +82,12 @@
 	@Value("${eureka.instance.hostname:${EUREKA_INSTANCE_HOSTNAME:}}")
 	String hostname;
 
+	@Value("${eureka.instance.statusPageUrlPath:}")
```

Comment:
Because the other values don't have camel case (`eureka.instance.hostname`, `management.port`, etc...) there are only two relaxed values and we account for those, this won't.